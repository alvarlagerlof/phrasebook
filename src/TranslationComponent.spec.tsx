import { render } from '@testing-library/react';
import * as Stories from './TranslationComponent.stories';

const { default: ignore, ...stories } = Stories;

describe('Translation component', () => {
  it.each(
    Object.entries(stories),
  )('%s', (storyName, storyFn) => {
    const { asFragment } = render(storyFn());

    expect(asFragment()).toMatchSnapshot();
  });
});
