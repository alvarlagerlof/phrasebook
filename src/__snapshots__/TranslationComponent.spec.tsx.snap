// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Translation component ComponentWithChildren 1`] = `<DocumentFragment />`;

exports[`Translation component Default 1`] = `<DocumentFragment />`;

exports[`Translation component OneParamTwoComponents 1`] = `<DocumentFragment />`;

exports[`Translation component TwoParamsOneComponent 1`] = `<DocumentFragment />`;
